adapters/repos/db/vector/hnsw/condensor.go

```diff
@@ -151,24 +151,24 @@ func (c *MemoryCondensor) SetLinksAtLevel(nodeid int, level int, targets []uint3
 			Warning("condensor length of connections would overflow uint16, cutting off")
 	}
 	ec.add(c.writeUint16(c.newLog, uint16(targetLength)))
-	ec.add(c.writeUint32Slice(c.newLog, targets[:targetLength]))
+	ec.add(c.writeUint64Slice(c.newLog, targets[:targetLength]))
 
 	return ec.toError()
 }
 
-func (c *MemoryCondensor) SetEntryPointWithMaxLayer(id int, level int) error {
+func (c *MemoryCondensor) SetEntryPointWithMaxLayer(id int64, level int) error {
 	ec := &errorCompounder{}
 	ec.add(c.writeCommitType(c.newLog, SetEntryPointMaxLevel))
-	ec.add(c.writeUint32(c.newLog, uint32(id)))
+	ec.add(c.writeUint64(c.newLog, uint64(id)))
 	ec.add(c.writeUint16(c.newLog, uint16(level)))
 
 	return ec.toError()
 }
 
-func (c *MemoryCondensor) AddTombstone(nodeid int) error {
+func (c *MemoryCondensor) AddTombstone(nodeid int64) error {
 	ec := &errorCompounder{}
 	ec.add(c.writeCommitType(c.newLog, AddTombstone))
-	ec.add(c.writeUint32(c.newLog, uint32(nodeid)))
+	ec.add(c.writeUint64(c.newLog, uint64(nodeid)))
 
 	return ec.toError()
 }
```

Comment:
as above, variables representing node ids should already be `uint64`, no longer requiring conversions